c3po/api/service/paginate.py

```diff
@@ -0,0 +1,39 @@
+from urllib.parse import  urlparse , urlunparse, parse_qs
+from collections import OrderedDict
+from urllib.parse import urlencode
+
+def get_paginated_response(posts, url, start=0, limit=24, offset=False):
```

Comment:
`offset` is for the queries that could go endlessly i.e. latest, popular, frequent. It is to avoid heavy queries to the DB

Reply:
Avoiding heavy queries should be the default (and only) behaviour rather than keeping it as an option.

Reply:
Gotcha! :+1: